spec/sources/buildkite_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'danger/ci_source/circle'
+
+describe Danger::CISource::Buildkite do
+  it 'validates when buildkite env var is found' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.